### PR TITLE
Add German Translation

### PR DESCRIPTION
**Pull request by infi**
I'm not sure what happened with the last line of README here, but it should be the same despite GitHub's `diff` saying otherwise.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ Participants asked to stop any harassing behavior are expected to comply immedia
 
 **Before** posting a message, take exactly 30 seconds to decide what channel is the best fit for that message. See the channel descriptions above.
 
-React with an emoji to acknowledge that you have read this message and agree to follow the code of conduct! :point_down:
\ No newline at end of file
+React with an emoji to acknowledge that you have read this message and agree to follow the code of conduct! :point_down:
```
Comment (snordmann) [inline README.md:60]: You added an extra new line at the end of the file.

File: README_GERMAN.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+>Hey! Wir brauchen deine Hilfe dabei, diese Nachricht in andere Sprachen zu übersetzen. Siehe [CONTRIBUTING](./CONTRIBUTING.md) für mehr details.
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
>Hey! Wir brauchen deine Hilfe dabei, diese Nachricht in andere Sprachen zu übersetzen. Siehe [CONTRIBUTING](./CONTRIBUTING.md) für mehr Details.
```

File: README_GERMAN.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+**Kurz gesagt** - **Lese diese gesamte Nachricht, oder du wirst nicht in der Lage sein, auf die restlichen Channels in diesem Server zuzugreifen.**
+
+Hallo Freund! Willkommen zum Coding Garden-Discord Server.
+
+Bevor du zu den anderen Channels zugrif bekommst, musst du dich in #🗣introductions (mit wenigstens 40 Zeichen) vorstellen und zeigen, dass du den Verhaltenskodex weiter unten gelesen hast, indem du unter dieser Nachricht mit einem Emoji reagierst 👇.
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
Bevor du zu den anderen Channels Zugriff bekommst, musst du dich in #🗣introductions (mit wenigstens 40 Zeichen) vorstellen und zeigen, dass du den Verhaltenskodex weiter unten gelesen hast, indem du unter dieser Nachricht mit einem Emoji reagierst 👇.
```

File: README_GERMAN.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Bevor du eine Nachricht absendest, nehme dir bitte 30 Sekunden Zeit, um zu entscheiden, in welchen Channel diese Nachricht kommt:
+**SENDE NICHT DIESELBE NACHRICHT IN MEHRERE CHANNELS.** Wie werden deine Nachricht sehen und antworten, wenn wir Zeit haben.
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
**SENDE NICHT DIESELBE NACHRICHT IN MEHRERE CHANNELS.** Wir werden deine Nachricht sehen und antworten, wenn wir Zeit haben.
```

File: README_GERMAN.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+>Hey! Wir brauchen deine Hilfe dabei, diese Nachricht in andere Sprachen zu übersetzen. Siehe [CONTRIBUTING](./CONTRIBUTING.md) für mehr details.
+
+**Kurz gesagt** - **Lese diese gesamte Nachricht, oder du wirst nicht in der Lage sein, auf die restlichen Channels in diesem Server zuzugreifen.**
+
+Hallo Freund! Willkommen zum Coding Garden-Discord Server.
+
+Bevor du zu den anderen Channels zugrif bekommst, musst du dich in #🗣introductions (mit wenigstens 40 Zeichen) vorstellen und zeigen, dass du den Verhaltenskodex weiter unten gelesen hast, indem du unter dieser Nachricht mit einem Emoji reagierst 👇.
+
+Bevor du eine Nachricht absendest, nehme dir bitte 30 Sekunden Zeit, um zu entscheiden, in welchen Channel diese Nachricht kommt:
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
Bevor du eine Nachricht absendest, nimm dir bitte 30 Sekunden Zeit, um zu entscheiden, in welchen Channel diese Nachricht kommt:
```

File: README_GERMAN.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+**SENDE NICHT DIESELBE NACHRICHT IN MEHRERE CHANNELS.** Wie werden deine Nachricht sehen und antworten, wenn wir Zeit haben.
+
+* Verpasse keine Livestreams mit #🔴live-chat 
+* Teile deine Arbeit in #🎨share_work 
+* Teile Tutorials oder Artikel die du nützlich gefunden hast in #📖share_resources
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
* Teile Tutorials oder Artikel, die du nützlich gefunden hast, in #📖share_resources 
```

File: README_GERMAN.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+* Finde jemanden zum Zusammenarbeiten in #👫find-a-pair 
+* Frage nach Hilfe für Code Katas/Challenges in #🤓code-kata-help 
+* Frage nach Hilfe zu spezifischen Videos/Streams in #📺video-help 
+* Frage nach allgemeiner Coding-Hilfe in #🌈general-help 
+* Frage nach genauer Coding-Hilfe in #🖥front-end-help bzw. #⏮back-end-help 
+* Frage CJ direkte Fragen in #🤔ask_me_anything (Falls es eine Hilfe-frage ist, ist vielleicht einer der anderen Channels besser geeignet)
+* Allgemeine Diskussion kommt in #💬general  
+* Alles, was Off-Topic oder ohne Topic ist, kann nach #🎲random 
+
+Code aus den Videos/Streams kann auf GitHub gefunden werden: <https://github.com/CodingGarden>
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
Der Quellcode aus den Videos/Streams kann auf GitHub gefunden werden: <https://github.com/CodingGarden>
```

File: README_GERMAN.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+Ein Patron mit einer monatlichen 5$-Spende werden: <https://www.patreon.com/CodingGardenWithCJ>
+
+Stickers und Merch können auf RedBubble gefunden werden: <https://www.redbubble.com/people/w3cj/collections/907384-coding-garden-with-cj>
+
+👇 Lese und bestätige die Verhaltenskodex-Nachricht weiter unten.
+
+**Code of Conduct**
+
+Indem du diesen Server betrittst oder Nachrichten schreibst, stimmst du dabei zu, dem Verhaltenskodex folge zu leisten. Alle Verstöße gegen diesen Verhaltenskodex können Konsequenzen wie einen Kick oder temporären Ban nach Ermessen der Moderatoren (Pruners). Sei nett!
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
Indem du diesen Server betrittst oder Nachrichten schreibst, stimmst du dabei zu, dem Verhaltenskodex folge zu leisten. Alle Verstöße gegen diesen Verhaltenskodex können Konsequenzen wie einen Kick oder temporären Ban nach Ermessen der Moderatoren (Pruners) haben. Sei nett!
```

File: README_GERMAN.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+  Verwende dein bestes Urteilsvermögen. Wenn es andere Unkomfortabel machen könnte, sende es nicht ab.
+
+* **Sei Respektvoll.** Meinungsverschiedenheiten sind keine Gelegenheit, die Gedanken oder Meinungen anderer anzugreifen. Auch wenn Meinungen sich underscheiden können, achte darauf, jede Situation mit Geduld und Pflege anzugehen. 
+* **Sei rücksichtsvoll.** Denke darüber nach, wie dein Beitrag in der Community ankommen wird. 
+* **Sei aufgeschlossen.** Umfasse neue Leute und neue Ideen. Unsere Community evolutioniert ständig, und positive Änderungen sind willkommen.
+
+Wenn du denkst, dass jemand gegen den Verhaltenskodex verstößt, bitten wir dich, das einem Moderator (Pruner) zu melden. Bitte gebe deinen Namen an, und eine Beschreibung des Vorfalls, und wir werden uns so schnell wie möglich zurückmelden.
+
+Teilnehmer, die gegen Richtlinien verstoßen, und gebeten werden, aufzuhören, sollten es sofort tun. Falls ein Teilnehmer sich belästigend Verhaltet, könnte es in Konsequenzen wie einem Kick oder temporären Ban nach Ermessen der Moderatoren erfolgen. Sei nett!
```
Comment (snordmann) [inline README_GERMAN.md]:
```suggestion
Teilnehmer, die gegen Richtlinien verstoßen, und gebeten werden, aufzuhören, sollten es sofort tun. Falls ein Teilnehmer sich belästigend verhält, könnte es in Konsequenzen wie einem Kick oder temporären Ban nach Ermessen der Moderatoren erfolgen. Sei nett!
```

**Review by snordmann — CHANGES_REQUESTED** [review]
Hi @spVierSechs 👋, thanks for your translation 👍 

I looked over it and fixed some minor spelling or grammatical errors.
I think we might need to reword the section `**Achte auf deine Wortwahl.**`; Especially the first bullet point has some improvement potential. I currently have no idea how to write it more concisely and still be broad enough to cover everything.

Other than that: Looks good! Thank you again for your work 😊

**Comment by infi** [comment]
The errors are mostly due to my insert key being broken, so I'm accidentally overwriting characters sometimes.  I currently have no idea how to merge your suggested changes, but I'll add them ASAP when I'm on my desktop PC again.  
  
CJ, if you're approving this, add the requested changes afterwards please.  
  
Again, I apologize for the spelling/grammar mistakes.

**Comment by w3cj** [comment]
No worries! Thanks so much to both of you for the contributions. I should be able to merge it all in.

**Comment by w3cj** [comment]
@spVierSechs It looks like I can't merge the changes directly. I'll wait for you to merge them and then I'll merge the pull request. No rush! Just leave a comment here when you've merged all the suggestions.

**Comment by infi** [comment]
Merged the suggestions.  
  
Edit: Looks like removing a newline using GitHub's web editor didn't help :( Other than that it's still the same content in README.md, so I think it's still fine to merge.

**Comment by w3cj** [comment]
Thanks for all your efforts!

I added a link in the README. Will you let me know if this is a good translation of:

>German: To read this message in german, click here.

>Deutsch: Um diese Nachricht auf Deutsch zu lesen, klicken Sie hier.

I used google translate.

**Pull request merged**